help/destinations/catalog/advertising/liveramp-onboarding.md

```diff
@@ -173,6 +179,8 @@ Once you've added all your desired mappings, select **[!UICONTROL Next]** and fi
 
 Your data is exported to the [!DNL LiveRamp - Onboarding] storage location that you configured, as CSV files.
 
+Exported files have a maximum size of 10 million rows, thus Adobe will generate multiple files per delivery if selected Segments exceed the 10MM row count.  Please let your LiveRamp representative know if you'll exceed a single file so that they can configure LiveRamp's ingestion automation to process the files in a batch fashion.
```

Comment:
Exported files have a maximum size of 10 million rows. Experience Platform generates multiple files per delivery if the selected audiences exceed 10 million rows.  If you expect to exceed the single file limit, contact your LiveRamp representative and ask them to configure batch ingestion for you.